orchestra/static/orchestra/main.es6.js

```diff
@@ -6,6 +6,7 @@ import {
 
 import common from 'orchestra/common/common.module.es6.js'
 import dashboard from 'orchestra/dashboard/dashboard.module.es6.js'
+import projectManagement from 'orchestra/project-management/project-management.module.es6.js'
```

Comment:
I don't feel strongly either way, just sticking to the convention I started—happy to change them all if you'd like!